gammapy/estimators/map/ts.py

```diff
@@ -511,7 +511,7 @@ def estimate_flux_map(self, datasets):
         j, i = zip(*positions)
 
         geom = maps[0]["counts"].geom.squash(axis_name="energy")
-        energy_axis = maps[0]["counts"].geom.axes["energy"]
+        energy_axis = geom.axes["energy"]
```

Comment:
This seems a cleanup.

Reply:
geom is squashed but not on the energy axis line